src/Header/Header.js

```diff
@@ -3,8 +3,35 @@ import "./Header.css";
 
 class Header extends React.Component {
   render() {
-    return <div>I am a top bar. please fix me</div>;
+    return (<div className="head">
+      <div className="greeting-row">
+        <span className="greeting">Welcome Username!</span>
+      </div>
+      <div className="button-row">
+        <div className="buttons">
+          <button
+            className="share"
+            onClick={() => alert('share link copied')}>
+            <span className="text1">
+              Share
+         </span>
```

Comment:
This closing tag and the one for the other span are one indent short of where they should be.